### PR TITLE
glDrawElements update indices description

### PR DESCRIPTION
**Pull request by ratchetfreak**
Make it explicit that it's an offset into the GL_ELEMENT_ARRAY_BUFFER

### CODE REVIEW

File: gl4/glDrawElements.xhtml [hunk 1]
```diff
@@ -94,7 +94,8 @@
             </dt>
             <dd>
               <p>
-                    Specifies a pointer to the location where the indices are stored.
+                    Specifies a offset of the first index in the array in the data store of the buffer currently bound to
```
Comment (fenollp) [inline gl4/glDrawElements.xhtml]: `a offset` -> `an offset`

Comment (ratchetfreak) [inline gl4/glDrawElements.xhtml]: fixed

**Comment by BSVino** [comment]
You used to be able to pass an actual pointer to some verts into this command, is that still true in core? If so we should update gl3/glDrawElements and gl4/glDrawElements to indicate that both are possible, and perhaps note that the offset is preferred.

(Sorry it took me so long to respond, crazy few days.)

**Comment by ratchetfreak** [comment]
Not in core. For example in the [4.4 core spec](https://khronos.org/registry/OpenGL/specs/gl/glspec44.core.pdf#subsection.10.3.9) page 323 there is the following line:

> If zero is bound to `ELEMENT_ARRAY_BUFFER` , the result of [glDrawElements et. al] is undefined.

**Comment by BSVino** [comment]
Works for me. Thanks!

**Pull request merged**